### PR TITLE
add maintenance completed service log

### PR DESCRIPTION
**Pull request by sulmer-redhat**

### CODE REVIEW

**Comment by sulmer-redhat** [comment]
@NautiluX @cattias

File: osd/maintenance_completed.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Info",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "{$DATE} Cluster maintenance completed",
+    "description": "{$DATE} cluster maintenance is now completed.",
```
Comment (cattias) [inline osd/maintenance_completed.json:6]: add "successfully"
and add that if the customer notices an issue they can open a support ticket

Comment (sulmer-redhat) [reply osd/maintenance_completed.json]: Done, can you please verify?

File: osd/maintenance_completed.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
     "severity": "Info",
     "service_name": "SREManualAction",
     "cluster_uuid": "${CLUSTER_UUID}",
-    "summary": "{$DATE} Cluster maintenance completed",
-    "description": "{$DATE} cluster maintenance is now completed.",
+    "summary": "{$DATE} Cluster maintenance successfully completed",
+    "description": "{$DATE} cluster maintenance is now successfully completed. If you require support, please file a support ticket",
```
Comment (cattias) [inline osd/maintenance_completed.json:6]: cluster maintenance is now successfully completed. Your cluster services are now fully back online, if you require any assistance from Red Hat please open a support ticket.

Comment (sulmer-redhat) [reply osd/maintenance_completed.json]: Done

**Pull request merged**